src/components/pagination/bl-pagination.css

```
@@ -0,0 +1,62 @@
+:host {
+  width: max-content;
+}
+
+.pagination {
+  --bl-pagination-main-color: var(--bl-color-secondary);
+  --arrow-disabled-color: var(--bl-color-content-passive);
+
+  display: flex;
+}
+
+.pagination * {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.page-container {
+  display: flex;
+  align-items: center;
+}
+
+.page-list {
+  display: flex;
+  align-items: center;
+  list-style: none;
+  user-select: none;
+}
+
+.dots {
+  margin: 7.5px;
+}
+
+.dots::before {
+  content: ' \B7 \B7 \B7';
+  color: var(--bl-color-secondary);
+}
+
+.pagination-helpers {
+  display: flex;
+  align-items: center;
+  margin-right: var(--bl-size-s);
+  gap: var(--bl-size-m);
+}
+
+.jumper,
+.select {
+  display: flex;
+  align-items: center;
+  gap: var(--bl-size-2xs);
+}
+
+label {
+  font-size: var(--bl-font-size-m);
+  font-weight: var(--bl-font-weight-medium);
```

```suggestion
  font: var(--bl-font-title-3-medium);
```

Currently you don't set font-family, so font is broken on stroybook.